### ISSUE TITLE
Any ability to send, e.g., ctrl+c as key entry?

### ISSUE THREAD

**Issue by kimonm** [Genymobile/scrcpy#389]

**Comment by rom1v** [comment]
> Is there a way to send control key combinations as keystrokes?

No, all Ctrl+key are captured by the client for its own actions.

A possibility would be to add a special key to escape the next sequence (like screen or tmux do, e.g. Ctrl+b, Ctrl+key).
